<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Display Area</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="ReceivedLabel">
      <property name="text">
       <string>Log Text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeft</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="LogText"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout2">
      <item>
       <widget class="QPushButton" name="AnswerButton">
        <property name="text">
         <string>Answer</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CallButton">
        <property name="text">
         <string>Call</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopButton">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
